1.14.6-kustomize-3.4.0/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM alpine:latest
+
+ENV K8SVERSION=1.14.6
+ENV RELEASEVER=2019-08-22
+ENV KUSTOMIZEVER=3.4.0
+
+RUN : build dependencies \
+ && apk --update --no-cache add ca-certificates python3 \
+ && apk --no-cache add --virtual build-dependencies curl openssl \
+ && pip3 install --upgrade pip awscli \
+ && : kubectl for eks \
+ && curl -O https://amazon-eks.s3-us-west-2.amazonaws.com/$K8SVERSION/$RELEASEVER/bin/linux/amd64/kubectl \
+ && curl -O https://amazon-eks.s3-us-west-2.amazonaws.com/$K8SVERSION/$RELEASEVER/bin/linux/amd64/kubectl.sha256 \
+ && openssl sha1 -sha256 kubectl \
+ && chmod +x ./kubectl \
+ && mv ./kubectl /usr/local/bin/kubectl \
+ && rm -rf kubectl.sha256 \
+ && : install aws-iam-authenticator \
+ && curl -O https://amazon-eks.s3-us-west-2.amazonaws.com/$K8SVERSION/$RELEASEVER/bin/linux/amd64/aws-iam-authenticator \
+ && curl -O https://amazon-eks.s3-us-west-2.amazonaws.com/$K8SVERSION/$RELEASEVER/bin/linux/amd64/aws-iam-authenticator.sha256 \
+ && openssl sha1 -sha256 aws-iam-authenticator \
+ && chmod +x ./aws-iam-authenticator \
+ && mv ./aws-iam-authenticator /usr/local/bin/aws-iam-authenticator \
```

Comment:
This is not required anymore, I think.

Reply:
:ha-n:
https://dev.classmethod.jp/cloud/aws/eks-update-get-token-cmd/

Thanks!

Reply:
fixed 😸